cypress/e2e/spec.cy.ts

```diff
@@ -0,0 +1,97 @@
+describe('Flow Tests', () => {
+  it('Shows the landing page', () => {
+    cy.visit('/')
+    cy.get('[data-testid=get_started_button]').click()
+  })
+
+  it('Navigates through the Medicaid only flow', () => {
+    const incomeName = "Suzy"
+    const incomeDescription = "Yardwork"
+    const incomeAmount = '55'
+    const expenseName = 'Gas'
+    const expenseDate = '09/04/2024'
+    const expenseAmount = '33'
```

Comment:
i'm thinking about "Places where we can improve the breadth of these tests." 
* could we do this as a random number? possibly also as a float (btw this should be a unit or component test, truly. esp with the floats since js+floats is 😬 )